scrapper.py

```diff
@@ -25,8 +29,8 @@ class Crawler:
     """
     Crawler implementation
     """
-    def __init__(self, seed_urls: list, max_articles: int):
-        pass
+    def __init__(self, seed_urls: list, max_articles: int=None):
```

Comment:
it will not be accepted - you cannot change interface. It was made only for lecture purposes.